### PR TITLE
README: add download section

### PR DESCRIPTION
**Pull request by aparcar**
The *Firmware Selector* was build to simplify users finding suitable
images for their devices. Let's give it more visibility so unleashing
your home router becomes an easier thing.

Signed-off-by: Paul Spooren <mail@aparcar.org>

### CODE REVIEW

**Comment by neheb** [comment]
/s/build/built

File: README.md [hunk 1]
```diff
@@ -11,6 +11,18 @@ full customization, to use the device in ways never envisioned.
...
+## Download
+
+Built firmware images are available for many architectures and come with a
+package selection to be used as WiFi home router. To quickly find a factory
+image usable to migrate from a vendor stock firmware to OpenWrt try the
+*Firmware Selector*.
+
+* [OpenWrt Firmware Selector](https://firmware-selector.openwrt.org/)
+
+If your device is supported please follow the **Info** link to see install
```
Comment (BKPepe) [inline README.md:23]:
```suggestion
If your device is supported, please follow the **Info** link to see the install
```

File: README.md [hunk 2]
```diff
@@ -11,6 +11,18 @@ full customization, to use the device in ways never envisioned.
 
 Sunshine!
 
+## Download
+
+Built firmware images are available for many architectures and come with a
+package selection to be used as WiFi home router. To quickly find a factory
+image usable to migrate from a vendor stock firmware to OpenWrt try the
```
Comment (BKPepe) [inline README.md:18]:
```suggestion
image usable to migrate from a vendor stock firmware to OpenWrt, try the
```

**Comment by aparcar** [comment]
@adschm do you have an opinion on this?

**Comment by adschm** [comment]
I'd personally not highlight just the firmware selector, but would link a general downloads landing page here.

This page then of course could advertise the firmware selector next to the other options to download images.

Apart from that detail, I actually (generally) expect to find download links for built stuff on a GitHub repo README, so I'm quite surprised we didn't offer that until now.

**Comment by aparcar** [comment]
@adschm do you mean https://downloads.openwrt.org/ or https://openwrt.org/downloads?

**Comment by adschm** [comment]
> @adschm do you mean https://downloads.openwrt.org/ or https://openwrt.org/downloads?

I'm not really sure, but it should be one of them.

**Comment by Ansuel** [comment]
> https://openwrt.org/downloads?

@adschm @aparcar @neheb I would use this as it looks more appealing... Why this still is not merged?

**Comment by neheb** [comment]
for one the description has a typo, build vs. built.

**Comment by neheb** [comment]
anyway, ACK on this commit.

**Comment by Ansuel** [comment]
@aparcar can you fix the typo so we can merge this?

**Comment by aparcar** [comment]
@Ansuel what missing typos?

**Comment by Ansuel** [comment]
@aparcar 

The Firmware Selector was **build** to simplify users finding suitable
images for their devices. Let's give it more visibility so unleashing
your home router becomes an easier thing.

Should be built i think

**Comment by aparcar** [comment]
@adschm I'd at least link both, the firmware selector and the general download page. From my understanding of usability one should go straight to a website with download after clicking on something called "downloads". Our current "downloads" page contains many many sub pages and a raw folder structure with targets/subtargets.

**Pull request merged**